dir.targets

```diff
@@ -73,6 +73,10 @@
     <BinPlaceConfiguration Include="netstandard">
       <RefPath>$(RefRootPath)netstandard/</RefPath>
     </BinPlaceConfiguration>
+    <!-- some libraries that produce packages will remain targeting netcoreapp2.0 -->
+    <BinPlaceConfiguration Condition="'$(BuildingNETCoreAppVertical)' == 'true'" Include="netcoreapp2.0">
```

Comment:
I chose this rather than cross-compiling those libraries for netcoreapp/netcoreapp2.0.  I'm thinking that we want to minimize the amount of cross-compiling and also don't want to bloat these packages by redistributing the assets that previously shipped, unless we have to.